<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arigin3DGui::CreatePoint</class>
 <widget class="QWidget" name="Arigin3DGui::CreatePoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point coordinate</string>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_CreatePoint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxVisibleItems">
      <number>16</number>
     </property>
     <item>
      <property name="text">
       <string>Coordinates</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fit center of gravity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Point</string>
      </property>
      <property name="icon">
       <iconset resource="../../Part/Gui/Resources/Part.qrc">
        <normaloff>:/icons/Part_Point_Parametric.svg</normaloff>:/icons/Part_Point_Parametric.svg</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_CreatePointMethod">
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Parameter</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QStackedWidget" name="widgetStack2">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page0_coordinate">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_X_Axis">
              <property name="text">
               <string>X:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_Y_Axis">
              <property name="text">
               <string>Y:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_Z_Axis">
              <property name="text">
               <string>Z:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="Gui::QuantitySpinBox" name="vertexX" native="true">
              <property name="unit" stdset="0">
               <string notr="true">mm</string>
              </property>
              <property name="value" stdset="0">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="Gui::QuantitySpinBox" name="vertexY" native="true">
              <property name="unit" stdset="0">
               <string notr="true">mm</string>
              </property>
              <property name="value" stdset="0">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="Gui::QuantitySpinBox" name="vertexZ" native="true">
              <property name="unit" stdset="0">
               <string notr="true">mm</string>
              </property>
              <property name="value" stdset="0">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>139</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page1_fit"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox_CreatePoint</tabstop>
  <tabstop>vertexX</tabstop>
  <tabstop>vertexY</tabstop>
  <tabstop>vertexZ</tabstop>
 </tabstops>
 <resources>
  <include location="../../Part/Gui/Resources/Part.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBox_CreatePoint</sender>
   <signal>activated(int)</signal>
   <receiver>widgetStack2</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
